File: seed/php-sdk/trace/src/Admin/Requests/AdminSendTestSubmissionUpdateRequest.php
<?php

namespace Seed\Admin\Requests;

use Seed\Core\Json\JsonSerializableType;
use Seed\Types\TestSubmissionUpdate;

class AdminSendTestSubmissionUpdateRequest extends JsonSerializableType
{
    /**
     * @var TestSubmissionUpdate $body
     */
    public TestSubmissionUpdate $body;

    /**
     * @param array{
     *   body: TestSubmissionUpdate,
     * } $values
     */
    public function __construct(
        array $values,
    ) {
        $this->body = $values['body'];
    }
}

File: seed/ruby-sdk-v2/trace/lib/seed/admin/types/admin_send_test_submission_update_request.rb
# frozen_string_literal: true

module Seed
  module Admin
    module Types
      class AdminSendTestSubmissionUpdateRequest < Internal::Types::Model
        field :submission_id, -> { String }, optional: false, nullable: false, api_name: "submissionId"
        field :body, -> { Seed::Types::TestSubmissionUpdate }, optional: false, nullable: false
      end
    end
  end
end
